hack/istio/install-music-store-demo.sh

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash
+
+##############################################################################
+# install-music-store-demo.sh
+#
+# Installs the kiali music store demo application
+# https://github.com/kiali/demos/tree/master/music-store
+# Works on both openshift and non-openshift environments.
+##############################################################################
+
+: ${CLIENT_EXE:=oc}
+: ${DELETE_DEMOS:=false}
+: ${MSTORE:=music-store}
+: ${BASE_URL:=https://raw.githubusercontent.com/kiali/demos/master}
+
+apply_network_attachment() {
+  NAME=$1
+cat <<NAD | $CLIENT_EXE -n ${NAME} apply -f -
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: istio-cni
+NAD
+    cat <<SCC | $CLIENT_EXE apply -f -
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: ${NAME}-scc
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  type: RunAsAny
+supplementalGroups:
+  type: RunAsAny
+users:
+- "system:serviceaccount:${NAME}:default"
+- "system:serviceaccount:${NAME}:${NAME}"
+SCC
+}
+
+install_mstore_app() {
+
+  if [ "${IS_OPENSHIFT}" == "true" ]; then
+    ${CLIENT_EXE} get project ${MSTORE} || ${CLIENT_EXE} new-project ${MSTORE}
+  else
+    ${CLIENT_EXE} get ns ${MSTORE} || ${CLIENT_EXE} create ns ${MSTORE}
+  fi
+
+  ${CLIENT_EXE} label namespace ${MSTORE} istio-injection=enabled --overwrite=true
+  ${CLIENT_EXE} apply -f https://raw.githubusercontent.com/leandroberetta/demos/master/music-store/ui.yaml -n ${MSTORE}
+  ${CLIENT_EXE} apply -f https://raw.githubusercontent.com/leandroberetta/demos/master/music-store/backend.yaml -n ${MSTORE}
+
+  ${CLIENT_EXE} wait --timeout 60s --for condition=available deployment/music-store-backend-v1 -n music-store
+  ${CLIENT_EXE} wait --timeout 60s --for condition=available deployment/music-store-ui-v1 -n music-store
+
+  export INGRESS_PORT=$(${CLIENT_EXE} -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="http2")].nodePort}')
+  if [ "${IS_OPENSHIFT}" == "true" ]; then
+       export INGRESS_HOST=$(crc ip)
+    else
+       export INGRESS_HOST=$(minikube ip)
+  fi
```

Comment:
`else` is indented in a different column from if/fi - this doesn't really matter in bash but it is confusing to read because the alignment is weird. Also, some of this is indented with 4 or 5 spaces, other times in these scripts the indentations are 2 spaces. Make your indentation consistent for better readability and debugging (harder to tell what code is where when the indentations are different throughout the code).

Here's suggestion for this portion, but I suggest going through these scripts and making the indentation consistent.

```suggestion
  if [ "${IS_OPENSHIFT}" == "true" ]; then
    export INGRESS_HOST=$(crc ip)
  else
    export INGRESS_HOST=$(minikube ip)
  fi
```